src/newbank/server/NewBankClientHandler.java

```diff
@@ -12,8 +12,8 @@
 public class NewBankClientHandler extends Thread {
 
     private NewBank bank;
-    private BufferedReader in;
-    private PrintWriter out;
+    private static BufferedReader in;
+    private static PrintWriter out;
```

Comment:
In theory, with the current architecture, we may have multiple handlers. Making the reader and writer class variables can have negative side effects. Also, the transfer will happen between accounts of customers belonging to the bank that is stored as an instance variable. We should not use class (static) methods to manipulate instance data.